### ISSUE TITLE
Accessibility: Column header menu is not accessible with the keyboard

### ISSUE THREAD

**Issue by talimarcus** [predixdesignsystem/px-data-grid#467]
### Steps to reproduce
1. Tab focus to the data-grid header

**Expected behavior:** The three vertical dots show up when the header has keyboard focus, and there is a way to open the menu (and select menu items) with the keyboard.
**Actual behavior:** The three vertical dots do not show up and it is impossible to know that the menu is even there.

### Version
px-data-grid: v1.1.4
Browsers: Chrome 64, Firefox 57, Safari 11

### Screenshot
**Vertical dots show up on mouse hover**
<img width="1002" alt="screen shot 2018-01-23 at 8 25 23 pm" src="https://user-images.githubusercontent.com/9092541/35314561-cad85e84-007b-11e8-9315-fcc566428b66.png">

**Vertical dots do not show up when header item has keyboard focus**
<img width="1000" alt="screen shot 2018-01-23 at 8 21 47 pm" src="https://user-images.githubusercontent.com/9092541/35314560-cabeb6c8-007b-11e8-90e2-ba4394b01ef9.png">

**Comment by alump** [comment]
Related #466

**Comment by limonte** [comment]
Sorting with the keyboard is now fixed, but I doubt that the column menu keyboard a11y is easily doable with the current implementation which depends on mouse events.

**Comment by alump** [comment]
Marking as enhancement, as action can be done with keyboard, but buttons are not what wanted.

**Comment by Aileen-H** [comment]
PRIORITY 2
